examples/samples/partitionedQueues.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  delay,
+  SendableMessageInfo,
+  QueueClient,
+  ReceiveMode,
+  generateUuid,
+  Namespace,
+  ReceiveHandler,
+  OnError,
+  OnMessage,
+  ServiceBusMessage,
+  MessagingError
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log(`str: ${str}`);
+console.log(`path: ${path}`);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock });
+
+  // populate the queue with messages
+  await sendMessages(client);
+
+  let rcvHandler: ReceiveHandler;
+
+  // retrieve messages from the queue
+  const onMessage: OnMessage = async (brokeredMessage: ServiceBusMessage) => {
+    console.log(
+      ` \n### Received message:
+        ID - ${brokeredMessage.messageId},
+        messageBody - ${brokeredMessage.body ? brokeredMessage.body.toString() : undefined},
+        partitionKey - ${brokeredMessage.partitionKey},
+        label - ${brokeredMessage.label}`
```

Comment:
The .Net sample talks about how the sequence number is affected in partitioned queues. Any idea what that is about and how we are showing the same here?

Reply:
SequenceNumber value is a unique 64-bit integer assigne4d  by ServiceBus. For partitioned queues, the topmost 16 bits reflect the partition id. Sequence numbers roll over to zero when the 48/64-bit range is exhausted. The usual the ascending SequenceNumber characteristics is no longer guaranteed as they depend on partitionId.

I think I should print the SequenceNumber as well to clear the confusion.

Reply:
In the latest commit, updated the print statement to print `SequenceNumber` in the received message.
And at the beginning of the code, added a comment to enable partitions in the azure portal for the topic.

This would clear the confusion ?

Reply:
Sure. Also add a comment on what the user is expected to see